deepspeed/runtime/lr_schedules.py

```diff
@@ -675,11 +675,16 @@ def __init__(self,
         self.warmup_type = warmup_type
         self.inverse_log_warm_up = 1.0 / math.log(self.warmup_num_steps)
         self.last_batch_iteration = last_batch_iteration
+        # Initialize lr in optimizer
```

Comment:
These changes are duplicated. Can we extract the code as a function and reuse it?